src/net/sourceforge/kolmafia/persistence/TCRSDatabase.java

```diff
@@ -674,22 +675,26 @@ private static void addEffectSource(
 
   private static void applyConsumableModifiers(
       final int usage, final String itemName, final TCRS tcrs) {
-    Integer lint = ConsumablesDatabase.getLevelReqByName(itemName);
-    int level = lint == null ? 0 : lint.intValue();
+    var consumable = ConsumablesDatabase.getConsumableByName(itemName);
+    Integer lint = ConsumablesDatabase.getLevelReq(consumable);
+    int level = lint == null ? 0 : lint;
```

Comment:
I haven't studied all usages of getLevelReqByName (which wraps getLevelReq), but I have to wonder why returning null, if it is not present, is better than returning 0.

Investigating/fixing that is not required for this PR.

Reply:
Good point. I'm not sure why it does that either